osu.Game/Overlays/Options/SliderOption.cs

```diff
@@ -104,13 +105,14 @@ public OsuSliderBar()
                         AutoSizeAxes = Axes.None,
                         RelativeSizeAxes = Axes.None,
                         Masking = true,
-                        BorderColour = new Color4(255, 102, 170, 255),
+                        BorderColour = OsuColor.SliderbarNub,
                         BorderThickness = 3,
                         Children = new[]
                         {
                             new Box
                             {
-                                Colour = new Color4(255, 102, 170, 0),
+                                Colour = new Color4(OsuColor.SliderbarNub.R,

```

Comment:
We'll call it Opacity, then. As for Grey, well, we already have a constructor that takes RGB, and imo it's pretty clear what a Color4 with RGB all set to the same value is trying to accomplish. An extension/helper method isn't necessary for it, we don't save much.